mirrord/config/src/config.rs

```diff
@@ -30,3 +30,7 @@ where
         self.unwrap_or_default().generate_config()
     }
 }
+
+pub trait FromMirrordConfig {
```

Comment:
We're planning on turning on "deny no docs" sooner rather than later, so please add docs for this.